client-src/elements/chromedash-ot-creation-page.ts

```diff
@@ -199,15 +199,29 @@ export class ChromedashOTCreationPage extends LitElement {
     );
   }
 
-  addHiddenFields() {
-    // Add a field for updating that an OT creation request has been submitted.
+  addAdditionalFields() {
+    // Add a hidden field for updating that an OT creation request has been submitted.
     this.fieldValues.push({
       name: 'ot_action_requested',
       touched: true,
       value: true,
       stageId: this.stage.id,
       alwaysHidden: true,
     });
+
+    // For non-prod environments, add a checkbox to bypass the checks ensuring
+    // that all values exist in Chromium files.
+    // TODO(DanielRyanSmith): Identify non-prod environments using a flag rather
+    // than the app's title.
+    if (this.appTitle !== 'Chrome Platform Status') {
```

Comment:
optional nit: again if this should be factored to settings so the flag for non-prod is consistent across the app